tests/test_render.py

```diff
@@ -2275,3 +2281,265 @@ def set_bc(a):
         self.wait_for_text_to_equal(
             'body', 'Error loading dependencies', timeout=2
         )
+
+    # Clientside tests
+    def run_value_assertions(self, assertions):
+        for id in assertions:
+            WebDriverWait(self.driver, 10).until(
+                EC.text_to_be_present_in_element_value(
+                    (By.ID, id),
+                    assertions[id]
+                ),
+                message='Failed assertion: #{}.value != {}'.format(
+                    id, assertions[id]
+                )
+            )
+
+
+    def test_simple_clientside_serverside_callback(self):
```

Comment:
I would like to see clientside callback tests in their own file

Reply:
I tried this, but it was compatible with our existing percy config  - https://github.com/plotly/dash-renderer/pull/143/commits/7482e414a7734373a62fd2be5c35b8d9f999339c

Reply:
Ok 😢